### PR TITLE
[DOCS-6865] Mobile Workspace Tasks

### PR DESCRIPTION
**Pull request by Amy-Hyland**
Adding Tasks feature documentation, so far.

### CODE REVIEW

File: mobile-workspace/latest/using/index.md [hunk 1]
```diff
@@ -133,3 +133,51 @@ You can move files within the Mobile Workspace.
 
     * The scan will be saved where you navigated.
     * If you do not wish to keep the scan, exit without saving. A dialog box will appear to confirm you want to discard the scan, press **Discard**. Press **Cancel** to return to save your scan.
+
+## Tasks
+
+You can view, filter, complete, add comments, and add attachments to tasks while logged into the Mobile Workspace.
```
Comment (gl-lovekesh) [inline mobile-workspace/latest/using/index.md:139]: Add attachment is not part of Goring release. User can view the  attachments.
Add attachment will be the part of Cholsey release.

File: mobile-workspace/latest/using/index.md [hunk 2]
```diff
@@ -133,3 +133,51 @@ You can move files within the Mobile Workspace.
...
+
+3. Press **Apply** to set the filter and **Reset** to reset the filter to default. The **X** in top right of the filter UI will close the window and return to the Task list.
+
+4. The filter chips will show the active filters and the refresh button will reset the filters.
+
+### Task view
+
+Within the Task view are details about the task: Task name, Due date, Priority, Assignee, Status, and Identifier. There is a back arrow to exit Task view and return to the Task list.
+
+Individual tasks have a “Comments” section below the task details. The most recent comment is shown with the user's icon, user name, date the comment was added, and the user’s comment. If a comment is too long, there is a **Read More** link. If there is more than one comment, you will see a link to **View all**. The link **Add a comment…** is below existing comments.
```
Comment (gl-lovekesh) [inline mobile-workspace/latest/using/index.md:170]: There is no read more option on long text now.

**Review by gl-lovekesh — CHANGES_REQUESTED** [review]

File: mobile-workspace/latest/using/index.md (hunk 1 above)
Comment (Amy-Hyland) [reply mobile-workspace/latest/using/index.md]: I've removed adding attachments part of the general tasks description.

File: mobile-workspace/latest/using/index.md (hunk 2 above)
Comment (Amy-Hyland) [reply mobile-workspace/latest/using/index.md]: I've removed that sentence.

**Review by gl-lovekesh — APPROVED** [review]

**Pull request merged**